metadata-ingestion/docs/sources/mysql/mysql_recipe.yml

```diff
@@ -12,9 +12,7 @@ source:
     # If you need to use SSL with MySQL:
     # options:
     #   connect_args:
-    #     ssl_ca: "path_to/server-ca.pem"
-    #     ssl_cert: "path_to/client-cert.pem"
-    #     ssl_key: "path_to/client-key.pem"
+    #     ssl_ca: "/etc/ssl/certs/ca-certificates.crt"
```

Comment:
we may keep original doc example as it is, since it is very generic
and instead adding a note explaining that `ssl_ca: "/etc/ssl/certs/ca-certificates.crt"` should usually work 

WDYT?